docs/builds/guides/integration/configuration.md

```diff
@@ -145,13 +145,13 @@ You can use the extended toolbar configuration format to access additional optio
     }
 ```
 
- * **`items`** &ndash; An array of toolbar item names. The components (buttons, dropdowns, etc.) which can be used as toolbar items are defined in `editor.ui.componentFactory` and can be listed using the following snippet: `Array.from( editor.ui.componentFactory.names() )`
+ * **`items`** &ndash; An array of toolbar item names. Most of the components (buttons, dropdowns, etc.) which can be used as toolbar items are described under {@link features/index Features} tab. Full list is defined in {@link module:ui/componentfactory~ComponentFactory editor.ui.componentFactory} and can be listed using the following snippet: `Array.from( editor.ui.componentFactory.names() )`. More details could be found in {@link framework/guides/creating-simple-plugin Creating a simple plugin}.
```

Comment:
We could use some new, more advanced "simple plugin" guide, as this one is kind of covered by existing plugins. And this kind of guide would be a good showcase for toolbar items etc.

Reply:
Yes, it could be helpful, but it seems like another ticket. Should I change something in the description above?

Reply:
Nah, the description is good - I'll make one small change to it I just noticed it needs. It's just an idea that already popped up somewhere. For now, what we have will do.